src/components/ConfirmationDialog/ConfirmationDialog.scss

```diff
@@ -1,39 +1,111 @@
 @import "constants/style";
 
-.confirmation-dialog__wrapper {
-  position: absolute;
-  top: 0;
-  left: 0;
-  display: flex;
-  min-width: 100%;
-  min-height: 100%;
-  align-items: center;
-  justify-content: center;
-  z-index: $backdrop-z-index;
-  background: $color-white;
-  padding: 0px 2rem;
+.confirmation-dialog__background {
+  position: fixed;
+  @include inset-0;
+  background-color: rgba($color: $color-backlog-blue, $alpha: 0.1);
+  backdrop-filter: blur(10px);
+}
+[theme="dark"] .confirmation-dialog__background {
+  background-color: rgba($color: $color-planning-pink, $alpha: 0.1);
 }
 
-@media screen and (max-width: 920px) {
-  .confirmation-dialog__wrapper {
-    padding: $padding--medium;
-  }
+.confirmation-dialog__wrapper {
+  position: fixed;
+  @include inset-0;
+  @include flex-center;
 }
 
 .confirmation-dialog {
-  max-width: 420px;
+  width: 360px;
   color: $color-black;
+  background-color: $color-white;
+  border-radius: $rounded--medium;
+  padding: $padding--medium;
+  border: 2px solid;
+  border-color: $color-backlog-blue;
+}
+[theme="dark"] .confirmation-dialog {
```

Comment:
personally, I prefer all dark theme stylings to be grouped at the bottom of the file, not scattered around like so